src/UI/walletConnect/WalletConnectLoginContainer/index.tsx

```diff
@@ -43,12 +48,31 @@ export const WalletConnectLoginContainer = ({
     logoutRoute,
     callbackRoute,
     token,
-    shouldLoginUser: true,
+    onLoginRedirect
+  });
+  const [
+    initLoginWithWalletConnectV2,
+    connectExisting,
```

Comment:
This break a bit the hook signature. If we return 3 items for all other hooks, the 4 items will break a common abstraction on top of them. Can we add this to the 3rd object, as an extra, specific config?